app/controllers/idv/image_uploads_controller.rb

```diff
@@ -20,16 +26,24 @@ def create
 
         if client_response.success?
           doc_pii_form_response = Idv::DocPiiForm.new(client_response.pii_from_doc).submit
-          form_response = form_response.merge(doc_pii_form_response)
+          analytics.track_event(
+            Analytics::IDV_DOC_AUTH_SUBMITTED_PII_VALIDATION,
+            doc_pii_form_response.to_h.merge(user_id: user_uuid),
+          )
           store_pii(client_response) if client_response.success? && doc_pii_form_response.success?
+
+          # merge in the image_form_response to pick up the remaining_attempts
+          doc_pii_form_response = image_form_response.merge(doc_pii_form_response)
         end
       end
 
-      analytics.track_event(Analytics::IDV_DOC_AUTH_SUBMITTED_IMAGE_UPLOAD_FORM, form_response.to_h)
-
       presenter = ImageUploadResponsePresenter.new(
         form: image_form,
-        form_response: presenter_response(form_response, client_response),
+        form_response: presenter_response(
+          image_form_response,
+          client_response,
+          doc_pii_form_response,
```

Comment:
consider making these keyword arguments so they're harder to mix up?

```suggestion
          image_form_response: image_form_response,
          client_response: client_response,
          doc_pii_form_response: doc_pii_form_response,
```